$OpenBSD: patch-dnssd_servicemodel_h,v 1.1 2018/04/18 19:29:46 naddy Exp $

The value of ServicePtrRole appears to be randomly chosen number
hat would be unlikely to collide with the values already used in
Qt::ItemDataRole.  It needs to fit into an int, so drop the top
bit.

Index: dnssd/servicemodel.h
--- dnssd/servicemodel.h.orig
+++ dnssd/servicemodel.h
@@ -71,7 +71,7 @@ class KDNSSD_EXPORT ServiceModel : public QAbstractIte
 
 	/** The additional data roles provided by this model */
 	enum AdditionalRoles {
-		ServicePtrRole = 0xA06519DE  ///< gets a RemoteService::Ptr for the service
+		ServicePtrRole = 0x206519DE  ///< gets a RemoteService::Ptr for the service
 	};
 
 	/**
